uui-components/src/inputs/NumericInput.tsx

```diff
@@ -77,19 +77,28 @@ const getFractionDigits = (formatOptions: Intl.NumberFormatOptions) => {
 };
 
 export const NumericInput = React.forwardRef<HTMLDivElement, NumericInputProps>((props, ref) => {
-    let {
-        value = null, min, max, formatOptions,
+    const {
+        value: initialValue = null,
+        min: initialMin,
+        max: initialMax,
+        formatOptions: initialFormatOptions,
+        step,
+        formatValue,
     } = props;
 
-    const { step, formatValue } = props;
+    const value = initialValue != null ? +initialValue : null;
+    const min = initialMin ?? 0;
+    const max = initialMax ?? Number.MAX_SAFE_INTEGER;
+    const formatOptions = initialFormatOptions ?? { maximumFractionDigits: 0 };
 
-    if (value != null) {
-        value = +value;
-    }
+    const placeholderValue = React.useMemo(() => {
+        if (!value && value !== 0) return props.placeholder || '0';
```

Comment:
Probably we can rewrite this condition to more straightforward: `if (value === null || value === undefined)`?